build/webpack.client.base.conf.js

```diff
@@ -7,11 +6,14 @@ var MiniCssExtractPlugin = require('mini-css-extract-plugin');
 var SvgStorePlugin = require('webpack-svgstore-plugin');
 var VueSSRClientPlugin = require('vue-server-renderer/client-plugin');
 
+const HardSourceWebpackPlugin = require('hard-source-webpack-plugin')
+const isProd = process.env.NODE_ENV === 'production';
+
 module.exports = merge.smart(baseWebpackConfig, {
 	entry: {
 		app: './src/client-entry.js'
 	},
-	devtool: config.build.productionSourceMap ? '#source-map' : false,
+	devtool: isProd ? 'source-map' : false,
```

Comment:
I'm fine with this. the '#' is webpack's default prefix for source map lines so I think it was unnecessary for our purposes.